Source/driver/Castro_setup.cpp

```diff
@@ -1089,91 +1089,44 @@ Castro::variableSetUp ()
   }
 #endif
 
-  // method of lines Butcher tableau
-  if (mol_order == 1) {
-
-      // first order Euler
-      MOL_STAGES = 1;
-
-      a_mol.resize(MOL_STAGES);
-      for (int n = 0; n < MOL_STAGES; ++n)
-        a_mol[n].resize(MOL_STAGES);
-
-      a_mol[0] = {1};
-      b_mol = {1.0};
-      c_mol = {0.0};
-
-  } else if (mol_order == 2) {
-
-    // second order TVD
-    MOL_STAGES = 2;
-
-    a_mol.resize(MOL_STAGES);
-    for (int n = 0; n < MOL_STAGES; ++n)
-      a_mol[n].resize(MOL_STAGES);
-
-    a_mol[0] = {0,   0,};
-    a_mol[1] = {1.0, 0,};
-
-    b_mol = {0.5, 0.5};
-
-    c_mol = {0.0, 1.0};
-
-  } else if (mol_order == 3) {
-
-    // third order TVD
-    MOL_STAGES = 3;
-
-    a_mol.resize(MOL_STAGES);
-    for (int n = 0; n < MOL_STAGES; ++n)
-      a_mol[n].resize(MOL_STAGES);
-
-    a_mol[0] = {0.0,  0.0,  0.0};
-    a_mol[1] = {1.0,  0.0,  0.0};
-    a_mol[2] = {0.25, 0.25, 0.0};
-
-    b_mol = {1./6., 1./6., 2./3.};
-
-    c_mol = {0.0, 1.0, 0.5};
-
-  } else if (mol_order == 4) {
-
-    // fourth order TVD
-    MOL_STAGES = 4;
-
-    a_mol.resize(MOL_STAGES);
-    for (int n = 0; n < MOL_STAGES; ++n)
-      a_mol[n].resize(MOL_STAGES);
-
-    a_mol[0] = {0.0,  0.0,  0.0,  0.0};
-    a_mol[1] = {0.5,  0.0,  0.0,  0.0};
-    a_mol[2] = {0.0,  0.5,  0.0,  0.0};
-    a_mol[3] = {0.0,  0.0,  1.0,  0.0};
-
-    b_mol = {1./6., 1./3., 1./3., 1./6.};
-
-    c_mol = {0.0, 0.5, 0.5, 1.0};
-
-  } else {
-    amrex::Error("invalid value of mol_order\n");
-  }
-
-
 
   if (sdc_order == 2) {
-
+    // Gauss-Lobatto (trapezoid)
     SDC_NODES = 2;
 
     dt_sdc.resize(SDC_NODES);
     dt_sdc = {0.0, 1.0};
 
   } else if (sdc_order == 4) {
-
+    // Gauss-Lobatto (Simpsons)
     SDC_NODES = 3;
 
     dt_sdc.resize(SDC_NODES);
     dt_sdc = {0.0, 0.5, 1.0};
 
+    //node_weights.resize(SDC_NODES);
+    //node_weights = {1.0/6.0, 4.0/6.0, 1.0/6.0};
+
+  } else if (sdc_order == 20) {
```

Comment:
Can we remove this Radau code from this PR please, and defer it to #666?

Reply:
done